pkg/serviced.default

```diff
@@ -126,7 +126,8 @@ SERVICED_FS_TYPE=btrfs
 # Default: 1    (enable)
 # SERVICED_MONITOR_DFS_MASTER_RESTART=1
 
+# Overrides the default for the service migration image.
+# SERVICED_SERVICE_MIGRATION_TAG=1.0.2
```

Comment:
As I understand it, all of those settings are commented out by default, and the user can uncomment if they want to enable it.   The variable itself is referenced at line 1526 in facade/service.go
```
	} else if tagOverride := os.Getenv("SERVICED_SERVICE_MIGRATION_TAG"); tagOverride != "" {
		tagName = tagOverride
	}
```